libdevcore/Base64.cpp

```diff
@@ -36,7 +36,11 @@ static inline bool is_base64(byte c)
 	return (isalnum(c) || (c == '+') || (c == '/'));
 }
 
+#ifdef __MIC__
+byte find_base64_char_index(byte c)
```

Comment:
What's the issue with this online?
Would `static byte find_base64_char_index(byte c)` be workable for both, and kill the ifdef?